ivy_tests/test_ivy/test_frontends/test_torchvision/test_ops.py

```diff
@@ -0,0 +1,145 @@
+# global
+import numpy as np
+from hypothesis import strategies as st
+
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+# --- Helpers --- #
+# --------------- #
+
+
+@st.composite
+def _nms_helper(draw):
+    img_width = draw(st.integers(250, 1250))
+    img_height = draw(st.integers(250, 1250))
+    num_boxes = draw(st.integers(5, 50))
+    bbox = {}
+    for _ in range(num_boxes):
+        x1 = draw(st.integers(0, img_width - 20))
+        w = draw(st.integers(5, img_width - x1))
+        y1 = draw(st.integers(0, img_height - 20))
+        h = draw(st.integers(5, img_height - y1))
+        bbox[(x1, y1, x1 + w, y1 + h)] = draw(st.floats(0.1, 0.7))
+    iou_threshold = draw(st.floats(0.2, 0.5))
+    return (
+        ["float32", "float32"],
+        np.array(list(bbox.keys()), dtype=np.float32),
+        np.array(list(bbox.values()), dtype=np.float32),
+        iou_threshold,
+    )
+
+
+@st.composite
+def _roi_align_helper(draw):
+    dtype = draw(helpers.get_dtypes("float", full=False))[0]
```

Comment:
Is this function only supposed to work with `float`? If so, please add `@with_supported_dtypes` and use `get_dtypes("valid", ...` here. It's a matter of good practice.

Reply:
I will change it.